src/module/macros.ts

```diff
@@ -396,6 +395,7 @@ export async function renderMacroTemplate(actor: LancerActor | undefined, templa
     terms: [],
     result: "",
   };
+  /*
```

Comment:
I'd rather just delete the code; it'll be in git history if we need it

Reply:
Whoops, I meant to. Fixed now